content/en/docs/Configuration/multi-cluster.md

```diff
@@ -34,10 +34,17 @@ The unified Kiali multi-cluster setup requires the Kiali Service Account (SA) to
    chmod +x kiali-prepare-remote-cluster.sh
    ./kiali-prepare-remote-cluster.sh --kiali-cluster-context east --remote-cluster-context west --view-only false
    ```
+{{% alert color="info" %}}
+If you wish to delete a remote cluster secret that was previously created by that script, pass in the command line option `--delete true` to the script along with the same command line options you passed in when creating the secret. Use the option `--help` for additional details on using the script to create and delete remote cluster secrets.
+{{% /alert %}}
```

Comment:
Rather than this and the other info alert being separate notes in the middle of the "setup" section, I think it would be easier for someone to find if removing a cluster was its own section like "cleanup" or just "removing a cluster" etc.

Reply:
> if removing a cluster was its own section like "cleanup" or just "removing a cluster" etc.

this is done